### ISSUE TITLE
App crashes after toggling "Show Mattermost icon in menu bar" setting

### ISSUE THREAD

**Issue by jasonblais** [mattermost/desktop#706]
I confirm (by marking "x" in the [ ] below: [x]):

- [X] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [X] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [X] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**
Mac: App crashes after toggling "Show Mattermost icon in menu bar" setting

**Steps to reproduce**

Mac; Mattermost desktop app v4.0.0; Mattermost server v4.6.0; 

1. Go to the Settings page via CMD+COMMA
2. Enable "Show Mattermost icon in menu bar" setting. Then disable it.

**Expected behavior**
Setting enabled, then disabled.

**Observed behavior**
App crashes

![image](https://user-images.githubusercontent.com/13119842/35838804-152fcb54-0abb-11e8-8ac2-d3ccf538d1c0.png)

Details:

```
Application: Mattermost 4.0.0
Platform: Darwin 16.7.0 x64
TypeError: Cannot read property 'setContextMenu' of null
    at EventEmitter.ipcMain.on (/private/var/folders/s7/wkz14w4s3vq__njkt9j5jrhc0000gn/T/AppTranslocation/DA6D1006-087E-4608-BAAF-FD2C60F3F716/d/Mattermost 6.app/Contents/Resources/app.asar/main_bundle.js:8214:15)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:194:7)
    at WebContents.<anonymous> (/private/var/folders/s7/wkz14w4s3vq__njkt9j5jrhc0000gn/T/AppTranslocation/DA6D1006-087E-4608-BAAF-FD2C60F3F716/d/Mattermost 6.app/Contents/Resources/electron.asar/browser/api/web-contents.js:247:13)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:194:7)
```

**Possible fixes**
?

**Comment by yuya-oc** [comment]
Closed via #720.

**Comment by jasonblais** [comment]
Thanks @yuya-oc!! :)
